geode-core/src/test/java/org/apache/geode/test/dunit/standalone/VersionManager.java

```diff
@@ -38,6 +38,9 @@
  */
 public class VersionManager {
   public static final String CURRENT_VERSION = "000";
+  public static final String GEODE_110 = "110";
+  public static final String GEODE_120 = "120";
```

Comment:
The `geode-old-versions/build.gradle` also fetches `1.0.0-incubating`, which gets filed under `"100"`.  As long as we're adding the aliases, we could add that one as well.

Also, someone left a commented-out `System.out.println` a few lines below this that I would love to see snipped.